### ISSUE TITLE
Enhancement

### ISSUE THREAD

**Issue by kmacmaster** [smeighan/xLights#663]
I see that the layout tab can be repositioned, very cool.  However, I would like to be able to float both the model / model group list and the properties box, out of the main window, and separate from each other.

For those with multiple monitors it would be nice get the list set at the full height of the screen as well as the properties.  There are a ton of properties that are by default all contracted, once expanded, having them full length would be a good use of screen real-estate.

**Comment by AzGilrock** [comment]
Looks like you already did.

**Comment by kmacmaster** [comment]
Remember when I said I was accustomed to the Photoshop way of doing things (layer discussion)?  Anyway, that is a Photoshopped mockup of what I am proposing.  If you look at the clocks on the two screens, you can see there was a few minutes put into this ;)

**Comment by AzGilrock** [comment]
Yeah I'm a Photoshop expert and a lot of the stuff in the program was modeled after Photoshop.  There are so many problems that arise once we start using the wxAUI manager to be able to have panels that can dock and float and remember their positions.  I'm not gonna make that change.  I can just hear the complaints about someone not being able to find their models window.

**Comment by kmacmaster** [comment]
Have you ever used Cakewalk or Sonar?  There is a dropdown in the corner of each pane that allows you to choose whether it should dock or float. That would be a useful tool that would solve the laptop issue of windows trying to auto-dock when you want them to float.  In this situation, it would save many people the hassle because they would have to choose for it to float.  However, I understand bugs and effects (functionality) over UI.

**Comment by AzGilrock** [comment]
If you hold down the control key when moving a panel it will no auto-dock so there is no issue to solve there.  By the way we need to be multi-platform and one of the ways we do that is by using wxWidgets which provides multi-platform base classes we use to design and operate the GUI.   You really shouldn't be spending a ton of time on your layout anyways.  Just get it drawn and start sequencing.  You rarely look at it once you get it set.  I have other users complaining about the time we spend on the Layout tab when it's such a small percentage of the time used in the program.  They want more content in other areas.  So no matter what we do it seems like some percentage of you is unhappy.

Also this was the worst title for an issue I've seen.  No one will have any idea what its about when they glance at the titles.

**Comment by kmacmaster** [comment]
Sorry, I think I started writing the title and forgot to finish it.  The post took me a few minutes since I was Photoshopping.  I don't think the multi-platform thing has anything to do with this issue, having float-able windows would be nice on all platforms.  I totally disagree with you on the layout tab.  I come from having a bunch of LOR channels I have not abandoned yet.  My layout alone has 14 RGBW AC trees which all need 4 layers to accomplish.  I have to spend some time in the layout tab to get it all right.  Any time spent making a product more user friendly is not wasted time.  Never said I was unhappy, thought it would be a "nice" feature since you have floating windows in the other areas.

A compromise... Outlook has the reading pane that can go on the bottom or to the right.  Maybe one day when the bug count gets low and you don't feel like doing math.  :)

**Comment by AzGilrock** [comment]
Sorry I don't do any enhancement where the user says they disagree with me. :)

**Comment by AzGilrock** [comment]
Note for developers:  We've talked about possibly trying to convert over to Qt.  That would be the right time to consider this design change.

**Comment by kmacmaster** [comment]
While I completely forgot about this post, I still think it's a move in the right direction. As the software and hobby have progressed, it seems more people are adding many more items than I probably had 8 years ago. But you're right, I'm usually too busy during Christmas to open xLights let alone try to do something in it other than test someone else's controller. Sigh.

**Comment by derwin12** [comment]
Gil .. could we make the right side a container that could be undocked - that might be "doable". Worth investigating.

**Comment by AGFazio** [comment]
Easy peasy --- PR 6090 - Intern does it again!!
